pkg/intel/metadata/common/manifestcodegen/pkg/analyze/scan.go

```diff
@@ -43,6 +43,7 @@ func getRawPkg(
 	buildCtx := build.Default
 	buildCtx.GOPATH = strings.Join(goPaths, string(filepath.ListSeparator))
 	buildCtx.BuildTags = append(buildCtx.BuildTags, `manifestcodegen`)
+	fmt.Printf("search for `%v` in `%v` (%v) \n\n", path, goPaths, buildCtx.GOPATH)
```

Comment:
Looks like leftover garbage :)

Reply:
it's from the debug commit - to be dropped once this whole thing is ready, being a draft as of now
I had other stuff to do... but now is an interesting time to get back to BoitGuard things, so here I am back on it again :)